website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step08-apply-helm-chart.sh

```diff
@@ -4,8 +4,4 @@ helm upgrade --install karpenter oci://public.ecr.aws/karpenter/karpenter --vers
   --set settings.aws.clusterName=${CLUSTER_NAME} \
```

Comment:
Can you add a comment above the `docker logout` line on why we logout. Something like

`# Logout of docker to perform an unauthenticated pull against the public ECR`

See: https://docs.aws.amazon.com/AmazonECR/latest/public/docker-pull-ecr-image.html